MagneticField/ParametrizedEngine/interface/alpaka/ParabolicParametrizedMagneticField.h

```diff
@@ -0,0 +1,45 @@
+/**
+ Description: Utility function to calculate the Magnetic Field on the GPU
+*/
+
+#ifndef DataFormats_EgammaReco_plugins_alpaka_MagneticFieldParabolicPortable_h
+#define DataFormats_EgammaReco_plugins_alpaka_MagneticFieldParabolicPortable_h
+
+#include <Eigen/Core>
```

Comment:
Eigen is not used in the code, please remove it.

Reply:
Removed.